labs/firepower-restapi-106/1.md

```diff
@@ -27,37 +28,27 @@ which you cannot edit, with the exception of security zones and interface groups
 
-You can group network, port, VLAN tag, URL, and PKI objects. Network object groups can be nested, that
-is, you can add a network object group to another network object group up to 10 levels. Objects and object groups of the same type cannot have the same name. In a multidomain deployment, the names of object groups must be unique within the domain hierarchy
+You can group network, port, VLAN tag, URL, and PKI objects. Network object groups can be nested; that is, you can add a network object group to another network object group up to 10 levels. Objects and object groups of the same type cannot have the same name. In a multi-domain deployment, the names of object groups must be unique within the domain hierarchy.
 
 ### Object overrides
-An object override allows you to define an alternate value for an object, which the system uses for the devices
-you specify.
-
-You can create an object whose definition works for most devices, and then use overrides to specify
-modifications to the object for the few devices that need different definitions. You can also create an object
-that needs to be overridden for all devices, but its use allows you to create a single policy for all devices.
-Object overrides allow you to create a smaller set of shared policies for use across devices without giving up
-the ability to alter policies when needed for individual devices.
-
-***For example:*** <mark> you might want to deny ICMP traffic to the different departments in your company, each of
-which is connected to a different network. You can do this by defining an access control policy with a rule
-that includes a network object called Departmental Network. By allowing overrides for this object, you can
-then create overrides on each relevant device that specifies the actual network where that device is connected. </mark>
+An object override allows you to define an alternate value for an object, which the system uses for the devices you specify.
+
+You can create an object whose definition works for most devices, and then use overrides to specify modifications to the object for the few devices that need different definitions. You can also create an object that needs to be overridden for all devices, but its use allows you to create a single policy for all devices. Object overrides allow you to create a smaller set of shared policies for use across devices, without giving up the ability to alter policies when needed for individual devices.
+
+***For example:*** <mark> You might want to deny ICMP traffic to the different departments in your company, each of which is connected to a different network. You can do this by defining an access control policy with a rule that includes a network object called Departmental Network. By allowing overrides for this object, you can then create overrides on each relevant device that specifies the actual network where that device is connected. </mark>
```

Comment:
I'm not sure what the `<mark>` tags do here? Does it do something in the output?